homeassistant/components/tradfri/const.py

```diff
@@ -7,7 +7,7 @@
 ATTR_SAT = "saturation"
 ATTR_TRADFRI_GATEWAY = "Gateway"
 ATTR_TRADFRI_GATEWAY_MODEL = "E1526"
-ATTR_TRADFRI_MANUFACTURER = "IKEA"
+ATTR_TRADFRI_MANUFACTURER = "IKEA of Sweden"
```

Comment:
This is the name we use in pytradfri so by changing here the device list will become more consistent.

Reply:
Are we using this const in state attributes anywhere or only in device info? Ie, is this a breaking change?

Reply:
It's being used when we add to the device registry:
https://github.com/home-assistant/home-assistant/blob/0235487a229a871010a45edc7d60ec583396e450/homeassistant/components/tradfri/__init__.py#L127